tools/demikernel-ci.py

```diff
@@ -230,21 +238,32 @@ def run_pipeline(
     step: int = 0
     status: int = 0
 
+    # Create folder for test logs
+    log_directory = "{}-{}-{}".format(libos, branch, "debug" if is_debug else "release")
```

Comment:
I would change this statement to:

```python
log_directory = "{}-{}-{}".format(libos, branch, "debug" if is_debug else "release").replace("/", "_")
```

This enables us to run the pipeline on detached target branches. This is useful when the target branch was forced pushed.

Reply:
Cool, will do